### PR TITLE
Updates: New blog post about /tools announcement

### PR DESCRIPTION
**Pull request by pbakaus**
This brings in the announcement of the new /tools hompage and the new home for DevTools.

### CODE REVIEW

**Comment by pbakaus** [comment]
@jpmedley can you take a look and approve if everything looks good? Meggin is on holiday, which is why I assigned it to you :)

**Comment by jpmedley** [comment]
Will do.

Joe Medley | Technical Writer, DevPlat | jmedley@google.com | 816-678-7195

File: src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown [hunk 1]
```diff
@@ -15,26 +15,37 @@ category: tools
...
+
+## DevTools' new home
+
+[![The new tools homepage](/web/updates/images/2015-08-07-google-web-tools/devtools.png)](/web/tools/chrome-devtools)
+
+The DevTools homepage – along with all documentation – moved from [https://developers.chrome.com](https://developer.chrome.com) to [https://developers.google.com/web/tools/chrome-devtools](/web/tools/chrome-devtools). But don't let the new shiny fool you: We have plenty of content additions and more substantial changes for you.
+
+## Action-oriented tutorials
+
+The DevTools docs that show you how to use DevTools are now goal- or action-oriented, similar to how [Web Fundamentals](http://developers.google.com/web/fundamentals) structures its pages. So instead of "Learn everything about the network panel", you'll get ["Measure Resource Loading Times"](/web/tools/profile-performance/network-performance/resource-loading).
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown]: Remove first occurrence of DevTools in this sentence.

File: src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown [hunk 2]
```diff
@@ -15,26 +15,37 @@ category: tools
...
+
+Going forward, our tutorials and guides will tell an overarching tooling story independent of any specific tool. What matters is that you're productive and efficient, and sometimes the combination of multiple tools gets you there. Tutorials from the old Tools section in Web Fundamentals have already been moved over and we'll work on adding more non-DevTools related content in the future to give you a full picture of developer productivity.
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown]: "have already been moved over" Delete the word over.

Comment (jpmedley) [inline src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown]: The last sentence is a bit much. Find a way to split it.

File: src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown [hunk 3]
```diff
@@ -15,26 +15,37 @@ category: tools
...
+
+Head over to the [Setup section](/web/tools/setup/) to get started.
+
+## Updates, news and useful tips
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown]: Comma after news.

File: src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown [hunk 4]
```diff
@@ -15,26 +15,37 @@ category: tools
...
+
+[![Tips and tricks](/web/updates/images/2015-08-07-google-web-tools/tips.png)](/web/updates/tools)
+
+Inspired by the fantastic [devtoolstips.com](http://devtoolstips.com/) and [Umar's DevTips](https://umaar.com/dev-tips/), and with the help and support of Umar, we've launched a new [updates section](/web/updates/tools) that features the latest DevTools news, tips and useful advice – as well as news from all of our other tooling projects.
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-05-google-s-web-tools-in-one-place.markdown]: comma after tips.

**Comment by jpmedley** [comment]
Back to you, Paul.

**Comment by pbakaus** [comment]
All fixed!

**Pull request merged**